internal/app/cloudwatch/usage.go

```diff
@@ -11,8 +11,12 @@ import (
 	aws_cloudwatch "github.com/aws/aws-sdk-go-v2/service/cloudwatch"
 	aws_cloudwatch_types "github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
 	converter "github.com/qonto/prometheus-rds-exporter/internal/app/unit"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/codes"
 )
 
+var tracer = otel.Tracer("")
```

Comment:
What will be the default name in that case for the tracer ?

Reply:
By default it's `go.opentelemetry.io/otel/sdk/tracer`. But as discussed, we'll use the package name generating the span which is helpful for troubleshooting.